pkg/policy/api/egress.go

```diff
@@ -148,12 +148,34 @@ type EgressRule struct {
 	//     - 'sg-XXXXXXXXXXXXX'
 	// +optional
 	ToGroups []ToGroups `json:"toGroups,omitempty"`
+
+	aggregatedSelectors EndpointSelectorSlice
+}
+
+// SetAggregatedSelectors creates a single slice containing all of the following
+// fields within the EgressRule, converted to EndpointSelector, to be stored
+// within the EgressRule for easy lookup while performing policy evaluation
+// for the rule:
+// * ToEntities
+// * ToCIDR
+// * ToCIDRSet
+// * ToFQDNs
+func (e *EgressRule) SetAggregatedSelectors() {
+	res := make(EndpointSelectorSlice, 0, len(e.ToCIDR)+len(e.ToFQDNs)+len(e.ToCIDRSet)+len(e.ToEndpoints)+len(e.ToEntities))
```

Comment:
Add a comment indicating why we are not appending `ToEndpoints` here.

Reply:
Added.